src/translations/es.translation

```diff
@@ -25,8 +25,12 @@
         <label>Establezca el campo Filtro, o el campo Filtro y el valor Filtro.</label>
     </customLabels>
     <customLabels>
+        <label><!-- Cannot Change Contact And Course Offering --></label>
+        <name>CannotChangeContactAndCourseOffering</name>
+    </customLabels>
+    <customLabels>
+        <label>No se puede eliminar el registro porque existen registros de relacionados. Elimine primero los registros secundarios.</label>
```

Comment:
@skosgi1 Sorry, one more thing. I just noticed this, it appears that the translation is changing here and I don't think we want that to happen. If possible, prefer to just have the label addition above.

Reply:
@robertbwatson  I Updated the file. Can you please take a quick look again?